portfolio/src/main/webapp/script.js

```diff
@@ -34,46 +46,43 @@ function createMap() {
 
 /** All setup code for markers. */
 function setupMarkers() {
-  locationsDict["Bean"] = {
-    m: null,
-    lat: 41.8826099,
-    lng: -87.6232902,
-    h: -7,
-    p: 22,
-    is:
-      '<p style="margin-top:-3px; margin-bottom:-3px" class="field-text">Cloud Gate</p>',
-    iw: null,
-  };
-  locationsDict["Skyline"] = {
-    m: null,
-    lat: 41.9147471,
-    lng: -87.6209924,
-    h: 185,
-    p: 17,
-    is:
-      '<p style="margin-top:-3px; margin-bottom:-3px" class="field-text">Photography Point</p>',
-    iw: null,
-  };
-  locationsDict["Riverwalk"] = {
-    m: null,
-    lat: 41.8877563,
-    lng: -87.6273952,
-    h: 125,
-    p: 30,
-    is:
-      '<p style="margin-top:-3px; margin-bottom:-3px" class="field-text">Chicago Riverwalk</p>',
-    iw: null,
-  };
-
-  for (var location in locationsDict) {
-    locationsDict[location].m = createMarker(locationsDict[location]);
-    locationsDict[location].iw = new google.maps.InfoWindow({
-      content: locationsDict[location].is,
+  locationsDict["Bean"] = new Location(
+    null,
+    41.8826099,
+    -87.6232902,
+    -7,
+    22,
```

Comment:
It can be difficult to match the value with the parameter. Comments will help. Follow the link below.

https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments